lib/stanford-mods/geo_spatial.rb

```diff
@@ -32,6 +32,22 @@ def geo_extensions_as_envelope
         []
       end
 
+      # @return [Array{String}] values suitable for solr SRPT fields, like "-16.0 28.0"
+      # @note example xml leaf nodes
+      #  <gml:pos>-122.191292 37.4063388</gml:pos>
+      def geo_extensions_point_data
+        mods_ng_xml.extension
+                   .xpath(
+                     '//rdf:RDF/rdf:Description/gmd:centerPoint/gml:Point[gml:pos]',
+                     'gml' => GMLNS,
+                     'gmd' => 'http://www.isotc211.org/2005/gmd',
+                     'rdf' => 'http://www.w3.org/1999/02/22-rdf-syntax-ns#'
+                   ).map do |v|
+                     lat, long = v.xpath('gml:pos', 'gml' => GMLNS).text.split
+                     "#{long} #{lat}"
+                   end
```

Comment:
And this morning, I wrote some tests for a variety of cases including:

- no `<extension>` elements
- no `<gmd:centerPoint>` element
- no `<gml:Point>` element
- no `<gml:pos>` element
- malformed data in the `<gml:pos>`, including schema-invalid data with too many elements, too few elements, or no text content at all.

Looking in #87, where the exception handling code for `geo_extensions_as_envelope` was added, the only test simulate an exception, but, again, it's unclear to me what real-world scenario we're guarding against.